project_AI/project_AI/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
```

Comment:
Now that Django is installed, your environment dependencies should be dumped into `requirements.txt` and included in this PR